userspace/libscap/scap.c

```diff
@@ -186,11 +186,13 @@ int32_t scap_init(scap_t* handle, scap_open_args* oargs)
 	if(strcmp(engine_name, SOURCE_PLUGIN_ENGINE) == 0)
 	{
 		vtable = &scap_source_plugin_engine;
+#ifdef __linux__
```

Comment:
Why is this needed? I mean, i think part of the @gnosek purpose when doing the platform refactor was to avoid conditionals like this one.
Also, this seems more like a fix unrelated to the PR, right?

Reply:
Seems like Grzegz implemented the same logic in https://github.com/falcosecurity/libs/blob/d1f3bd3a5d51b89e90030ff666967f866a1e49f9/userspace/libscap/scap.c#L38C50-L38C50. This is a leftover, good catch! Let me remove it.